apps/snitch_core/lib/core/data/schema/state.ex

```diff
@@ -19,5 +19,6 @@ defmodule Snitch.Data.Schema.State do
     |> cast(attrs, [:abbr, :name, :country_id])
     |> validate_required([:abbr, :name, :country_id])
     |> foreign_key_constraint(:country_id)
+    |> unique_constraint(:abbr, name: :index_states_on_abbr_and_country_id)
```

Comment:
unique_constraint needs to be applied to one field, even though the index it refers to spans multiple fields.Here we are applied only on :abbr

Reply:
Please add a message that explains this constraint.